src/main/io/ledstrip.c

```diff
@@ -494,8 +495,13 @@ static const struct {
     {0,             LED_MODE_ORIENTATION},
 };
 
+ledBarState_t ledBarStates[MAX_BAR_STATES];
+
 static void applyLedFixedLayers(void)
 {
+    ledBarStates[LED_BAR_GPS].count = 0;
+    ledBarStates[LED_BAR_BATTERY].count = 0;
```

Comment:
Stack-allocated variable will do
```suggestion
    uint8_t ledBarIdx[MAX_BAR_STATES] = {0};
```